invenio_previewer/__init__.py

```diff
@@ -234,78 +246,61 @@
 
 1. By providing a ``previewer`` key in the file, with the name of the previewer
```

Comment:
Could we clarify this a bit more? maybe with a code snippet? saying, "adding a key in the file" sounds really abstract to me and I wouldn't know how to do it TBH :see_no_evil:.